operator/redisfailover/service/generator.go

```diff
@@ -110,6 +111,10 @@ func generateRedisConfigMap(rf *redisfailoverv1alpha2.RedisFailover, labels map[
 	namespace := rf.Namespace
 
 	labels = util.MergeLabels(labels, generateLabels(redisRoleName, rf.Name))
+	redisConfigFileNameContent := `slaveof 127.0.0.1 6379
```

Comment:
I'd rename this to `redisConfigFileContent`

Reply:
I'll change it 😊